packages/flutter/test/widgets/single_child_scroll_view_test.dart

```diff
@@ -107,6 +107,49 @@ void main() {
     expect(context.clipBehavior, equals(Clip.hardEdge));
   });
 
+  testWidgets('SingleChildScrollView ScrollViewKeyboardDismissBehavior.onDrag test',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: SingleChildScrollView(
+            keyboardDismissBehavior: ScrollViewKeyboardDismissBehavior.onDrag,
+            child: Column(
+              children: <Widget>[
+                Autocomplete<String>(
+                  optionsBuilder: (TextEditingValue textEditingValue) {
+                    return <String>['One', 'Two', 'Three'].where((String option) {
+                      return option.contains(textEditingValue.text.toLowerCase());
+                    }).toList();
+                  },
+                  fieldViewBuilder: (BuildContext context, TextEditingController textEditingController, FocusNode focusNode, VoidCallback onFieldSubmitted) {
+                    return TextField(
+                      controller: textEditingController,
+                      focusNode: focusNode,
+                    );
+                  },
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+    final Finder textFieldFinder = find.byType(TextField);
+    expect(textFieldFinder, findsOneWidget);
+
+    await tester.tap(textFieldFinder);
+    await tester.pump();
+
```

Comment:
Can you add an expectation that the options are visible here, and then one that they are not visible at the end of the test? I guess you could just check for the text 'One'.